1-org/envs/shared/README.md

```diff
@@ -18,7 +18,8 @@
 | dns\_hub\_project\_budget\_amount | The amount to use as the budget for the DNS hub project. | `number` | `1000` | no |
 | domains\_to\_allow | The list of domains to allow users from in IAM. Used by Domain Restricted Sharing Organization Policy. Must include the domain of the organization you are deploying the foundation. To add other domains you must also grant access to these domains to the terraform service account used in the deploy. | `list(string)` | n/a | yes |
 | enable\_hub\_and\_spoke | Enable Hub-and-Spoke architecture. | `bool` | `false` | no |
-| enable\_os\_login\_policy | Enable OS Login Organization Policy. | `bool` | `false` | no |
+| enable\_os\_login\_policy | Enable OS Login Organization Policy. | `bool` | `true` | no |
```

Comment:
Let's not expose this as a var anymore unless there is user demand.

Reply:
This was something that Andy asked on the meeting where he presented us these new org policies. 

The requested change for this partitular "os_login_policy" was for us to swap this variable default value to be "true" (enabled) and then show the user that he can change it to false in case he fell into one of the limitations situations that this constraint might bring up to a particular scenario.

Could you please double check wheter we should keep or not this var for this case?